src/routes/rootCatalog.ts

```diff
@@ -25,9 +25,9 @@ const selfLinks = (req: Request): Link[] => {
     },
     {
       rel: "service-desc",
-      href: `${stacRoot}/resources/swagger.json`,
+      href: `https://api.stacspec.org/v1.0.0-beta.1/openapi.yaml`,
       title: "OpenAI Documentation",
```

Comment:
Should this be:
```suggestion
      title: "OpenAPI Documentation",
```

Reply:
Great catch so used to seeing openAI these days didn't look weird